client/src/components/categorical/categorical.js

```diff
@@ -127,119 +135,65 @@ class Categories extends React.Component {
           padding: globals.leftSidebarSectionPadding
         }}
       >
+        <AnnoDialog
+          isActive={createAnnoModeActive}
+          title="Create new category"
+          instruction="New, unique category name:"
+          cancelTooltipContent="Close this dialog without creating a category."
+          primaryButtonText="Create new category"
+          text={newCategoryText}
+          validationError={this.categoryNameError(newCategoryText)}
+          errorMessage={this.categoryNameErrorMessage(newCategoryText)}
+          handleSubmit={this.handleCreateUserAnno}
+          handleCancel={this.handleDisableAnnoMode}
+          annoInput={
+            <AnnoInputs
+              text={newCategoryText}
+              handleItemChange={this.handleSuggestActiveItemChange}
+              handleChoice={this.handleChoice}
+              handleTextChange={this.handleNewCategoryText}
+            />
+          }
+          annoSelect={
+            <AnnoSelect
+              handleModalDuplicateCategorySelection={
+                this.handleModalDuplicateCategorySelection
+              }
+              categoryToDuplicate={categoryToDuplicate}
+              allCategoryNames={allCategoryNames}
+            />
+          }
+        />
+
         {/* READ ONLY CATEGORICAL FIELDS */}
         {/* this is duplicative but flat, could be abstracted */}
-        {_.map(allCategoryNames, catName =>
-          !schema.annotations.obsByName[catName].writable ? (
-            <Category
-              key={catName}
-              metadataField={catName}
-              createAnnoModeActive={createAnnoModeActive}
-              isUserAnno={false}
-            />
-          ) : null
+        {_.map(
```

Comment:
we have moved away from lodash where native JS replacements exist.  In this case, please replace with:  `allCategoryNames.map(...)`

Reply:
and remove the lodash import from this file.